src/platform/common/activeEditorContext.ts

```diff
@@ -37,7 +37,9 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
     private isJupyterKernelSelected: ContextKey;
     private hasNativeNotebookOrInteractiveWindowOpen: ContextKey;
     constructor(
-        @inject(IInteractiveWindowProvider) private readonly interactiveProvider: IInteractiveWindowProvider,
+        @inject(IInteractiveWindowProvider)
+        @optional()
+        private readonly interactiveProvider: IInteractiveWindowProvider,
```

Comment:
this type should say `IInteractiveWindowProvider | undefined`. Makes the code easier later if it's explicit about it being undefined.